### PR TITLE
Adding new option, summaryOnly

### PR DESCRIPTION
**Pull request by illumination517**
Adding new flag as discussed in https://github.com/gruntjs/grunt-contrib-qunit/issues/116.

### CODE REVIEW

**Comment by vladikoff** [comment]
Is it possible to add a test for the output?

**Comment by illumination517** [comment]
I'll take a look and try and workout how we test for that in the morning.

File: docs/qunit-options.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ Default: `false`
 
 When true, the whole task will not fail when there are individual test failures, or when no assertions for a test have run. This can be set to true when you always want other tasks in the queue to be executed.
 
+## summaryOnly
+Type: `boolean`  
+Default: `false`
+
+When true, this will suppress the all default logging for individually failed tests. Customized logging can be performed by listening to and responding to `qunit.log` events.
```
Comment (Arkni) [inline docs/qunit-options.md]: I think that `... all ...` can be omitted.

File: docs/qunit-options.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ Default: `false`
 
 When true, the whole task will not fail when there are individual test failures, or when no assertions for a test have run. This can be set to true when you always want other tasks in the queue to be executed.
 
+## summaryOnly
+Type: `boolean`  
+Default: `false`
```
Comment (Arkni) [inline docs/qunit-options.md]: You should add this option to the default options (line `158 => 170`), otherwise its default value is `undefined`

**Comment by illumination517** [comment]
Hey guys, I've added some updates in from code review. I can't see an easy way to add a test for this feature. Any suggestions?

**Comment by vladikoff** [comment]
Looks like we don't test the output at this time anyway, I'll merge it but if anyone wants to get better testing for this plugin please send a PR.

**Comment by vladikoff** [comment]
Waiting for ` continuous-integration/appveyor/pr `

**Comment by Arkni** [comment]
Looks good to me.
Can you squash the commits into 1 ?

**Comment by illumination517** [comment]
Done :)

**Comment by illumination517** [comment]
One more quick question, when will this be published to NPM? :)

**Comment by vladikoff** [comment]
@Arkni should we release 1.0.2 or 1.1?

**Comment by Arkni** [comment]
I'll go with `1.1.0` as this PR added a new feature.

**Comment by Arkni** [comment]
@illumination517 published as `1.1.0`

**Comment by illumination517** [comment]
Thanks guys :)

**Pull request merged**